<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Experiments</title>
    <link href="./idec-css/reset123.css" rel="stylesheet" />
    <link href="./idec-css/experiments.css" rel="stylesheet" />
    <!-- <link href="./bootstrap.min.css" rel="stylesheet"> -->
</head>

<body>
    <header>
        <div class="nav-container">
            <div class="container">
                <div class="logo ">
                    <a class="brand" href="home.html"><img src="assets/img/logo.png" alt=""><img
                            src="./images/teamlogo.png" alt=""
                            style="width: 100px; vertical-align: middle; margin-top: -10px;"></a>
                </div>
                <div class="menu clearfix">
                    <div class="menu-active1 rightfix">
                        <ul class="menu-list "><a href="home.html">HOME</a></ul>
                        <ul class="menu-list "><a href="#">TEAM <i class="fa fa-angle-down" aria-hidden="true"></i></a>
                            <li><a href="member.html">MEMBERS</a></li>
                            <li><a href="attributions.html">ATTRIBUTIONS</a></li>
                        </ul>
                        <ul class="menu-list "><a href="#">PROJECT <i class="fa fa-angle-down"
                                    aria-hidden="true"></i></a>
                            <li><a href="background.html">BACKGROUND</a></li>
                            <li><a href="design.html">DESIGN</a></li>
                            <li><a href="Results.html">RESULTS</a></li>
                            <li><a href="Report.html">REPORT</a></li>
                           
                            <li><a href="supplement.html">SUPPLEMENT</a></li>
                            <li><a href="poster.html">POSTER</a></li>
                        </ul>
                        <ul class="menu-list "><a href="#">DOCUMENTATION <i class="fa fa-angle-down"
                                    aria-hidden="true"></i></a>
                            <li><a href="experiments.html">EXPERIMENTS</a></li>
                            <li><a href="safety.html">SAFETY</a></li>
                            <li><a href="protocol.html">PROTOCOL</a></li>
                        </ul>
                    </div>
                </div>
            </div>
        </div>
    </header>
    <img id="back-to-top-button" src="images/back top.png" width="50px" alt="返回顶部">
    <div class="Intermediate-content">
        <div class="content-row">
            <div class="content-row-list">
                <div class="row-cols-1">
                    <div class="content-col-4">
                        <img src="./images/baniere-gauche2.png" alt="">
                    </div>
                    <div class="content-col-4-h1">
                        <h1>EXPERIMENTS</h1>
                    </div>
                    <div class="content-col-4">
                        <img src="./images/baniere-droite1.png" alt="">
                    </div>
                </div>
            </div>
            <div class="content-summary_accordion">

                <div class="summary">
                    <div id="sticky_summary">
                        <div class="bubble" id="bubble_1">
                            <a href=""></a>
                            <h2>NNU CHINA IN APRIL </h2>

                        </div>

                        <div class="vert_summary"></div>
                        <div class="bubble" id="bubble_2">
                            <a href=""></a>
                            <h2>NNU CHINA IN MAY</h2>

                        </div>
                        <div class="vert_summary"></div>
                        <div class="bubble" id="bubble_3">
                            <a href=""></a>
                            <h2>NNU CHINA IN JUNE</h2>
                            <i></i>
                        </div>
                        <div class="vert_summary"></div>
                        <div class="bubble" id="bubble_4">
                            <a href=""></a>
                            <h2>NNU CHINA IN JULY</h2>
                            <i></i>
                        </div>
                        <div class="vert_summary"></div>
                        <div class="bubble" id="bubble_5">
                            <a href=""></a>
                            <h2>NNU CHINA IN AUGUST</h2>
                            <i></i>
                            <!-- <i>IN SITU</i> -->
                        </div>
                        <!-- <div class="vert_summary"></div> -->
                    </div>
                </div>

                <div class="accordion accordion_exp" id="accordionExample">

                    <div class="accordion-item" id="item_1" onclick="growCircle(1)">
                        <h2 class="accordion-header h2_exp">
                            <button class="accordion-button">
                                <p>
                                    Construct Plasmid and Search for Suitable Deep Eutectic Solvent
                                </p>
                            </button>

                        </h2>
                        <div class="accordion-collapse" id="item_1_collapse" style="display:none;">
                            <div class="accordion-body">
                                <h3>1. Obtain pet-22b ,BSLA and Bs2Est fragment by PCR</h3>
                                <figure>
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig0.png">
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig1.png">
                                    <figcaption>

                                        <center align="justify" style="font-size: 18px;"><b>Figure 1.</b>
                                            The Enzyme digestion products are detected by agarose gel
                                            electrophoresis, and the correct target fragment is 5476 bp. We obtained
                                            the
                                            correct
                                            target fragment, and the sample is purified.
                                        </center>

                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <h3>2. Gibson assembly</h3>


                                <ol>
                                    <li>Procedure:</li>
                                </ol>
                                <p style="text-indent: 2em;text-align: justify;"><b>a.</b> The above reaction
                                    system was incubated
                                    at 50 °C for 15 minutes
                                    (inserting 1-2
                                    fragments), 30 minutes (inserting 3 fragments simultaneously), or 60 minutes
                                    (inserting 4-5 fragments simultaneously). If subsequent operations cannot be
                                    carried
                                    out immediately after the reaction is completed, the reaction sample can be
                                    stored
                                    at -20 °C. </p></br>
                                <ol>
                                    <li>Transformation and cloning identification:</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;"><b>b.</b> Take 5uL of the reaction sample
                                    from the
                                    previous step and add it
                                    to receptive
                                    cells such as 50-100 uL of DH5a (note that the volume of the added DNA sample
                                    should
                                    not exceed 1/10 of the receptive volume). Gently mix and place the mixture on
                                    ice
                                    for 30 minutes.</p>
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> Heat shock in a 42 °C water bath
                                    for 90
                                    seconds, then quickly return to the ice
                                    bath and let stand for 3-5 minutes.
                                    c. Add 500 uL of antibiotic free SOC or LB culture medium, gently mix well, and
                                    shake
                                    at 37 °C for 1 hour.</p>
                                <p align="justify" style="text-indent: 2em;"><b>d.</b> Centrifuge the bacterial solution
                                    at
                                    5000 g for 1 minute to precipitate the
                                    bacterial body. Suck off most of the culture medium, leaving about 50-100 uL of
                                    culture medium, and resuspend the bacterial body. Then apply all evenly onto LB
                                    plates containing appropriate antibiotics and incubate overnight in a 37 °C
                                    incubator.</br></p>
                                <p align="justify" style="text-indent: 2em;"><b>e.</b> The next day, select clones from
                                    the
                                    plate for colony PCR or extract plasmids
                                    for
                                    enzyme digestion identification, or directly select several clones for
                                    sequencing
                                    identification.</p>



                                <figure>
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig2.png">
                                    <figcaption>
                                        <center  style="font-size: 18px;"><b>Figure 2.</b>
                                            The pet-22b fragments and BSLA, Bs2Est are connected by Gibson assembly, and
                                            the system is as follows.
                                        </center>
                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <figure>
                                    <div class="saut_1_ligne"></div>
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig3.png">
                                    <figcaption>
                                        <center style="font-size: 18px;"><b>Figure 3.</b>
                                            The pet-22b fragments and BSLA, Bs2Est are connected by Gibson assembly

                                        </center>
                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <h3>3. Selection of DES solutionR</h3>

                                <ol>
                                    <li>Pre cultivation of microorganisms:</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;"><b>a.</b> Cultivation of Escherichia coli:
                                    Use a needle to pick up WT and culture them in
                                    shake
                                    tubes with 150 μL amp-resistant LB liquid culture medium at 37 °C,900 rpm for 24
                                    hours.</p>
                                <div class="saut_1_ligne"></div>
                                <ol>
                                    <li>Main cultivation of microorganisms:</li>
                                </ol>

                                <p align="justify" style="text-indent: 2em;"><b>b.</b> Add 2 μL precultured bacterial
                                    solution to 150 μL automatic induction culture
                                    medium,
                                    culture at 37 °C, 900 rpm for 16 hours.</p>
                                Preparation of DES solvent:
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> TEA buffer:triethanolamine (50mM,
                                    pH 7.4) </p>
                                <p align="justify" style="text-indent: 2em;"><b>d.</b> 95% choline chloride and ethylene
                                    glycol (1:2): choline chloride 139.62g +
                                    ethylene
                                    glycol 124.14 g, heat and stir at 80 °C, and diluted to 95% with TEA buffer</p>
                                <p align="justify" style="text-indent: 2em;"><b>e.</b> 30% choline chloride and
                                    acetamide (1:2): choline chloride 139.62 g + acetamide
                                    118.14 g , heat and stir at 8 °C, and diluted to 30% with TEA buffer</p>
                                <p align="justify" style="text-indent: 2em;"><b>f.</b> 30% tetrabutylammonium bromide
                                    and ethylene glycol (1:2): tetrabutylammonium
                                    bromide
                                    339.33 g + ethylene glycol 124.14 g , heat and stir at 80 °C, and diluted to 30%
                                    with
                                    TEA
                                    buffer</p>
                                <p align="justify" style="text-indent: 2em;"><b>g.</b> 75% choline chloride and ethylene
                                    glycol (1:2): choline chloride 139.62 g +
                                    ethylene
                                    glycol 124.14 g , heat and stir at 80 °C, and diluted to 75% with TEA buffer</p>
                                </p>

                                <figure>
                                    <img alt="photo" class="w-80 mx-auto d-block" src="./images/fig4.png" style="width: 600px;">
                                    <div class="saut_1_ligne"></div>
                                    <figcaption>
                                        <center style="font-size: 18px;"><b>Figure 4.</b>
                                            Screening system of BSLA in three DESs.
                                        </center>
                                        </p>
                                    </figcaption>
                                </figure>
                                <p align="justify">
                                    (a)-(c) Residual activity of BSLA WT at different DES concentrations.</br>
                                    (b)The
                                    standard deviation to evaluate the applicability of the 96-well MTP-based
                                    screening system for directed BSLA WT evolution. From left to right, the
                                    compounds are 95% (v/v) ChCl:ethylene glycol, 30% (v/v) ChCl:acetamide, and
                                    30% (v/v) TBPB:ethylene glycol, which used for the screening system to
                                    obtain residual activity of 30-40% of the BSLA WT.</br>
                                <div class="saut_1_ligne"></div>
                                <figure>
                                    <img alt="photo" class="w-80 mx-auto d-block" src="./images/fig5.png"style="width: 600px;>
                                    <figcaption>
                                        <center style="font-size: 18px;"><b>Figure 5.</b>
                                            Determine the solution for the high-throughput screening system.
                                        </center>
                                    </figcaption>
                                </figure>
                                <p align="justify">After
                                    excluding the DES types that tend to solidify at room temperature or are not
                                    easy to handle with high viscosity, we used choline chloride and ethylene
                                    glycol in a 1:2 ratio to configure concentrations of the solution.</p>
                            </div>
                        </div>
                    </div>
                    <div class="gap_accordion"></div>
                    <div class="accordion-item" id="item_2" onclick="growCircle(2)">
                        <h2 class="accordion-header h2_exp">
                            <button class="accordion-button">
                                <p>Establish the BSLA and Bs2Est Mutant Library</p>
                            </button>

                        </h2>
                        <div class="accordion-collapse" id="item_2_collapse" style="display:none;">
                            <div class="accordion-body">

                                <h3>1. Extraction of plasmid DNA </h3>
                                <ol>
                                    <li>Extracting bacterial DNA with a Axygen kit for extracting and
                                        purifying DNA</li>
                                </ol>
                                <div class="saut_1_ligne"></div>
                                <h3>2. PCR</h3>
                                <ol>
                                    <li> <a href="./images/Primers used for corner engineering of the BSLA gene in pet.pdf"
                                            style="color:#589f7d;">Click Here </a>to obtain
                                        primers used for corner engineering of the BLSA gene in pet-22</li>
                                </ol>

                                <div class="saut_1_ligne"></div>
                                <h3>3. Agarose gel electrophoresis </h3>
                                <ol>
                                    <li>Preparation:</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;"><b>a.</b> 10X DNA Loading buffer</p>
                                <p align="justify" style="text-indent: 2em;"><b>b.</b> 15000 bp DNA Marker</p>
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> PCR product</p>
                                <p align="justify" style="text-indent: 2em;"><b>d.</b> Nucleic acid gel dye(1 g agarose
                                    +100mL 1X TAE)</p>
                                <p align="justify" style="text-indent: 2em;"><b>e.</b> Buffer solution for nucleic acid
                                    gel electrophoresis()dilute 50X TAE to 1X with
                                <p align="justify" style="text-indent: 2em;"><b>f.</b> RO</p>

                                <div class="saut_1_ligne"></div>
                                <ol>
                                    <li>Usage process:</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;"><b>a.</b> Add 100 mL 1X TAE to 1 g agar
                                    powder
                                    and add it into the conical flask specially
                                    made
                                    for agarose gel.</p>
                                <p align="justify" style="text-indent: 2em;"><b>b.</b> Boil in the microwave three times
                                    until completely dissolved, cool until not too
                                    hot
                                    to touch (45-55 °C), add 10 uL of nucleic acid dye, shake gently, and pour into the
                                    prepared mold, with a thickness of 60mm. Wait for it to solidify, then pull out the
                                    comb
                                    vertically to avoid damaging the sample hole.</p>
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> Store the prepared nucleic acid
                                    gel in the lunch box.</p>
                                <p align="justify" style="text-indent: 2em;"><b>e.</b> Add 1X TAE buffer solution to the
                                    electrophoresis tank, and place the prepared gel
                                    run
                                    (large hole recovery gel and small hole verification gel, with the gel hole located
                                    at
                                    the negative electrode), with the buffer solution not passing through the sample
                                    hole.</p>
                                <p align="justify" style="text-indent: 2em;"><b>f.</b> Sampling: Select the appropriate
                                    size marker, mix the sample with the loading
                                    buffer,
                                    and then load the sample.</p>
                                <p align="justify" style="text-indent: 2em;"><b>g.</b> Set the voltage and press "run"
                                    to
                                    run the glue for about 30 minutes.</p>
                                <p align="justify" style="text-indent: 2em;"><b>h.</b> Take out the nucleic acid gel and
                                    observe it with a light microscope.</p>
                                <div class="saut_1_ligne"></div>
                                <ol>
                                    <li>Precautions:</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;"><b>a.</b> Pay attention to protection and
                                    wear gloves when operating!</p>
                                <p align="justify" style="text-indent: 2em;"><b>b.</b> To make glue, dilute TAE should
                                    be used, heated and cooled to 45 °C before adding
                                    nucleic acid dye. The prepared glue can be stored at 4 °C for one week.</p>
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> The adhesive runs from the
                                    negative electrode to the positive electrode.</p>

                                <figure>
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig7.png">
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig8.png">
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig9.png">
                                    <figcaption>
                                        <center style="font-size: 18px;"><b>Figure 6.</b> A part of nucleic acid gel
                                        </center>
                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <h3>4. Transformation of receptive cells</h3>
                                <p align="justify" style="text-indent: 2em;"><b>a.</b> Take 100 µL of thawed receptive
                                    cells on ice, add the target plasmid, gently mix
                                    well, and let stand on ice for 30 minutes. </p>
                                <p align="justify" style="text-indent: 2em;"><b>b.</b> Heat shock the sample in a 42 °C
                                    water bath for 45-60 seconds, transfer it
                                    quickly to an ice bath, and let it stand for 2 minutes (do not shake the sample
                                    during the process of standing on ice, otherwise the conversion efficiency will be
                                    reduced). </p>
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> Add 700 µL of sterile liquid
                                    medium (SOB or LB) without antibiotics to the
                                    centrifuge tube, mix well, and resuscitate at 37 °C and 200 rpm for 60 minutes. </p>
                                <p align="justify" style="text-indent: 2em;"><b>d.</b> According to the needs of the
                                    experiment, draw different volumes of
                                    resuscitation fluid and evenly spread it on the SOB or Lysogeny broth containing
                                    corresponding antibiotics, and put the plate upside down in the 37 °C incubator for
                                    overnight culture. </p>


                                <div class="saut_1_ligne"></div>
                                <h3>5. Select strains for pre cultivation</h3>
                                <p align="justify" style="text-indent: 2em;">The bacterial liquid was transferred to the
                                    liquid LB supplemented with 1% 100
                                    mg/mL Amp, at the same time.</br>
                                    Pre cultivation system: 200 μL liquid LB + 1% 100 mg/mL Amp +bacterial broth
                                </p>
                                <div class="saut_1_ligne"></div>
                                <h3>6. Main culture</h3>
                                <p align="justify" style="text-indent: 2em;">2 μL pre cultured bacterial solution, add
                                    self induced culture medium, culture them in
                                    4 mL LB shake tubes with amp-resistant properties at 37 °C, 900 rpm for 16 hours.
                                </p>
                                <div class="saut_1_ligne"></div>
                                <h3>7. Freeze storage of glycerol bacteria</h3>
                                <p align="justify" style="text-indent: 2em;"><b>a.</b> Generally, the bacterial solution
                                    used for seed preservation needs to be thicker,
                                    so the cultivation time needs to be overnight;</p>
                                <p align="justify" style="text-indent: 2em;"><b>b.</b> Under the condition of Aseptic
                                    technique, add 500 uL bacterial solution and 500 uL
                                    50% glycerol solution into the sterilized 2 mL cryopreservation tube respectively,
                                    mix gently and fully;</p>
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> Glycerol cryopreserved bacteria
                                    can be directly stored in a -80 °C
                                    refrigerator.The frozen bacteria can be stored under this condition for several
                                    years. Subsequent freezing and thawing operations will reduce the shelf life.</p>
                                <p align="left">(Note: Items that come into
                                    contact with the bacterial solution, such as gun heads,
                                    cryotubes, and 50% glycerol solution, must be in a sterile state and ensure that the
                                    operation is carried out under sterile conditions.)</p>
                                <div class="saut_1_ligne"></div>
                                <h3>8. Enzyme activity measurement</h3>
                                <ol>
                                    <li>The usage process of the enzyme-linked immunosorbent assay instrument</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;"><b>a.</b> Turn on the computer and enzyme
                                    marker power;</p>
                                <p align="justify" style="text-indent: 2em;"><b>b.</b> Place the enzyme label plate
                                    correctly. Do not put it in with a cover to prevent
                                    it from getting stuck inside the instrument; If the sample is accidentally spilled,
                                    it must be cleaned in a timely manner; Do not touch the card slot area when
                                    transferring the enzyme label board to prevent hand injuries;</p>
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> Open the software and set
                                    parameters such as wavelength, temperature, and
                                    oscillation frequency;</p>
                                <p align="justify" style="text-indent: 2em;"><b>d.</b> Select the scanning hole
                                    position;</p>
                                <p align="justify" style="text-indent: 2em;"><b>e.</b> Conduct a scan, and the
                                    instrument reading is relatively accurate between 0.4
                                    and 1.0, with an upper limit of 4.0, if the value is too large, dilute it before
                                    proceeding with the measurement; After scanning, save the data in a personal
                                    folder;</p>
                                <p align="justify" style="text-indent: 2em;"><b>f.</b> Remove the enzyme marker board
                                    and turn off the device.</p>
                            </div>
                        </div>
                    </div>
                    <div class="gap_accordion"></div>
                    <div class="accordion-item" id="item_3" onclick="growCircle(3)">
                        <h2 class="accordion-header h2_exp">
                            <button class="accordion-button">
                                <p>The Mutants Obtained in the First Round were Rescreened for a Second Round</p>
                            </button>

                        </h2>
                        <div class="accordion-collapse" id="item_3_collapse" style="display:none;">
                            <div class="accordion-body">
                                <h3>1.Pre-work </h3>

                                <p align="justify" style="text-indent: 2em;"><b>a.</b> Pre-culture experiment content:
                                    Toothpicks were dipped into glycerol bacteria in 96-well plates, in which four
                                    parallel experiments were done for each mutant, and they were precultured with
                                    LBamp. Take 150 μL from each well in the glycerol plate into the medium in
                                    microtiter
                                    plate, (37°C, 70% humidity, 900rpm, 24h) the obtained pre-culture solution can be
                                    used for the subsequent main culture experiments.</p>
                                <p align="justify" style="text-indent: 2em;"><b>b.</b> The main culture experiment
                                    content:
                                    96-well plate as a container, take 2uL of pre-culture solution in the modified
                                    auto-induced Amp medium (components) to express protein, (37 °C, 70% humidity, 900
                                    rpm, 16 hours).</p>
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> Enzyme activity assay:
                                    The culture was centrifuged (4°C, 2000 rpm, 20 min) to obtain the crude enzyme,
                                    which was used for further enzyme activity assay. The esterase activity of BSLA and
                                    Bs2Est was determined using p-nitrophenyl butyrate (pNPB) as substrate. 95 μL
                                    DES/Buffer and 5 μL WT(Variants)/EV were incubated for 2 h at room temperature, and
                                    then the A410 nm was measured on the microtiter plate reader (Biotek Synergy HI,
                                    USA), and the release of pNP was recorded in 8 minutes.</p>

                                <div class="saut_1_ligne"></div>
                                <h3>2.Week 1 </h3>
                                <figure>
                                    <img alt="photo" class="w-80 mx-auto d-block" src="./images/fig10.png">
                                    <figcaption>
                                        <p align="justify">
                                            <center style="font-size: 18px;"><b>Figure 7.</b> A total of 7positions
                                                (P5/V6,V9/H10,A15/S16,K35/L36,T47/N48,S28/Q29,K88/N89,H76/S77)
                                                were screened for beneficial mutations of BSLA. Beneficial variants have
                                                been
                                                preserved in glycerol 96-well plates in previous experiments and can be
                                                used
                                                for a
                                                second round of screening.
                                            </center>
                                        </p>
                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <h3>3.Week 2 </h3>
                                <figure>
                                    <img alt="photo" class="w-80 mx-auto d-block" src="./images/fig11.png">
                                    <figcaption>
                                        <p align="justify">
                                            <center style="font-size: 18px;"><b>Figure 8.</b> A total of 8
                                                positions(L108/T109,S130/S131,M137/N138,I123/L124,I157/G158,Y161/S162/S163,L173/N174,L102/G103)
                                                were screened for beneficial mutations of BSLA. Beneficial variants have
                                                been preserved in glycerol 96-well plates in previous experiments and
                                                can be
                                                used for a second round of screening.
                                            </center>
                                        </p>
                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <h3>4.Week 3 </h3>
                                <figure>
                                    <img alt="photo" class="w-80 mx-auto d-block" src="./images/fig12.png">
                                    <figcaption>
                                        <p align="justify">
                                            <center style="font-size: 18px;"><b>Figure 9.</b> A total of 8 positions
                                                (P33-V34, E128-V129,
                                                D155-N156,
                                                K205-G206, M213-E214, M221-T222, E286-E287, and R38-F39) were screened
                                                for
                                                beneficial mutations of Bs2Est. Beneficial variants have been preserved
                                                in
                                                glycerol 96-well plates in previous experiments and can be used for a
                                                second
                                                round of screening.</center>
                                        </p>
                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <h3>5.Week 4 </h3>
                                <figure>
                                    <img alt="photo" class="w-80 mx-auto d-block" src="./images/fig13.png">
                                    <figcaption>
                                        <p align="justify">
                                            <center style="font-size: 18px;"><b>Figure 10.</b> A total of 7 positions
                                                (F143-L144, S375-H376,
                                                Q294-G295, R349-S350, R137-L138, L401-E402, and I421-T422) were screened
                                                for
                                                beneficial mutations of Bs2Est. The mutants were assayed for activity
                                                according to the pre-culture, main culture and enzyme activity assay
                                                methods
                                                described previously, and the data were collated.
                                            </center>
                                        </p>
                                    </figcaption>
                                </figure>
                            </div>
                        </div>
                    </div>
                    <div class="gap_accordion"> </div>
                    <div class="accordion-item" id="item_4" onclick="growCircle(4)">
                        <h2 class="accordion-header h2_exp">
                            <button class="accordion-button">
                                <p>BSLA and Bs2Est Expression and Purification</p>
                            </button>
                        </h2>
                        <div class="accordion-collapse" id="item_4_collapse" style="display:none;">
                            <div class="accordion-body">
                                <h3>1.Week 1</h3>
                                <ol>
                                    <li>Prepared the medium:</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;">LB-medium: Yeast extract 5 g/L, Tryptone
                                    10 g/L, NaCl 10 g/L</p>
                                <p align="justify" style="text-indent: 2em;">Autoinduction medium:
                                    The media include: media components, buffer components and induction components</p>
                                <figure>
                                    <table class="center-table">
                                        <tr>
                                            <th>Composition</th>
                                            <th>Weight</th>
                                        </tr>
                                        <tr>
                                            <td>Peptone (aus Casein, Roth)</td>
                                            <td>12 g</td>
                                        </tr>
                                        <tr>
                                            <td>Yeast extract (Merck)</td>
                                            <td>24 g</td>
                                        </tr>
                                        <tr>
                                            <td>Glycerol (Roth)</td>
                                            <td>5 g</td>
                                        </tr>
                                        <tr>
                                            <td colspan="2">Add dd. H<span style="font-size: x-small;">2</span>O to 800
                                                mL</td>
                                        </tr>
                                    </table>
                                    <center style="font-size: 18px;"><b>Figure 11.</b> Media components,more mediums <a
                                            href="images/mediums.pdf"
                                            style="color: #0F5132;font-weight: 700;"><strong>Click
                                                Here</strong></a></br>
                                    </center>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <ol>
                                    <li>Protein Induction Expression:</li>
                                </ol>
                                <figure>
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig15.png">
                                   
                                    <div class="saut_1_ligne">  </div>
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig16.png">
                                    
                                    <div class="saut_1_ligne">  </div>
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig17.png">
                                    <figcaption>
                                        <center align="justify" style="font-size: 18px;"><b>Figure 12.</b> SDS-PAGE.Based on
                                            the protein gel analysis,
                                            it can be
                                            observed that both 24 hours and 48 hours of induction result in satisfactory
                                            protein expression levels. Considering all factors, it is recommended to
                                            choose a 24-hour induction period for optimal protein expression.
                                            According to the gel plot analysis, BSLA was expressed in both LB and
                                            self-induction medium, with auto-induction medium showing better expression.
                                            Therefore, we chose the auto-induction medium to induce the expression of
                                            BSLA.

                                        </center>
                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <ol>
                                    <li>Purification:</li>
                                </ol>
                                Preparing Buffer Solutions with Different Imidazole Concentrations
                                <p align="justify" style="text-indent: 2em;"><b>a.</b> 50 mM potassium phosphate buffer
                                    (pH 8.0)
                                    1000 mL</p>
                                <p align="justify" style="text-indent: 2em;"><b>b.</b> 50 mM sodium phosphate buffer,
                                    300 mM
                                    sodium chloride, and 20 mM imidazole, pH
                                    8.0</p>
                                <p align="justify" style="text-indent: 2em;"><b>c.</b> 50 mM sodium phosphate buffer,
                                    300 mM
                                    sodium chloride, and 50 mM imidazole, pH
                                    8.0</p>
                                <p align="justify" style="text-indent: 2em;"><b>d.</b> 50 mM sodium phosphate buffer,
                                    300 mM
                                    sodium chloride, and 100 mM imidazole, pH
                                    8.0</p>
                                <p align="justify" style="text-indent: 2em;"><b>e.</b> 50 mM sodium phosphate buffer,
                                    300 mM
                                    sodium chloride, and 250 mM imidazole, pH
                                    8.0</p>
                                <p align="justify" style="text-indent: 2em;"><b>f.</b> 50 mM sodium phosphate buffer,
                                    300 mM
                                    sodium chloride, and 500 mM imidazole, pH
                                    8.0</p><br>

                                Use Ni-NTA 6FF(Pre-Packed Gravity Column) to purified BSLA and Bs2Est. Selecting 250
                                mM Imidazole as the Final Elution Concentration
                                </p>
                                <div class="saut_1_ligne"></div>
                                <h3>2.Week 2</h3>
                                <ol>
                                    <li>Inducing Expression of BSLA and Bs2Est Mutant in LB Medium</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;">Toothpick dip the bacterial liquid into 4
                                    ml LB: First, use a
                                    toothpick to transfer the bacterial liquid into a tube containing 4 mL of LB medium.
                                    LB medium is a nutrient-rich medium commonly used for culturing bacteria such as
                                    Escherichia coli. 37 °C, 220 rpm, 12 hours: Place the tube containing the bacterial
                                    liquid in a constant temperature shaker and culture it at 37 °C and 220 rpm for 12
                                    hours. This step aims to allow the bacterial liquid to
                                    grow and proliferate under suitable temperature and agitation conditions. Inoculate
                                    1% volume of the cultured liquid into 100 mL LB, 37 °C, 220 rpm, 2 hours: Take out
                                    1%
                                    (by volume) of the liquid cultured in the previous step and add it to a large
                                    container containing 100 mL of auto-induction medium. Then, continue to culture it
                                    at 30°C for 16h. </p>
                                <div class="saut_1_ligne"></div>
                                <ol>
                                    <li>Toothpick dip the bacterial liquid into 4 ml LB</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;">First, use a toothpick to transfer the
                                    bacterial liquid into a tube containing 4 mL
                                    of LB medium. LB medium is a nutrient-rich medium commonly used for culturing
                                    bacteria such as Escherichia coli. 37 °C, 220 rpm, 12 hours: Place the tube
                                    containing
                                    the bacterial liquid in a constant temperature shaker and culture it at 37 °C and
                                    220 rpm for 12 hours. This step aims to allow the
                                    bacterial liquid to grow and proliferate under suitable temperature and agitation
                                    conditions. Inoculate 1% volume of the cultured liquid into 100 mL LB, 37 °C, 220
                                    rpm,
                                    2 hours: Take out 1% (by volume) of the liquid cultured in the previous step and add
                                    it to a large container containing 100 mL of LB medium.</p>
                                <p align="justify" style="text-indent: 2em;">Then, continue to culture it
                                    at 37 °C and 220 rpm for 2 hours. This step aims to
                                    scale up the volume of the bacterial liquid to provide more cells for expression.
                                    0.1 mM IPTG, 20 °C, 180 rpm, 24 hours: Add IPTG (Isopropyl
                                    β-D-1-thiogalactopyranoside)
                                    at a concentration of 0.1 millimolar to the culture medium to induce the expression
                                    of the target protein. Next, continue to culture it at 20 °C and 180
                                    rpm for 24 hours. IPTG is a synthetic compound that can mimic
                                    signals bacteria receive in their natural environment, thus triggering the
                                    expression of the target protein. E. coli cells were harvested by centrifugation at
                                    11600 g for 10 min at 4 °C, and then washed twice with proper buffer (pH 7.5). The
                                    washed cells were resuspended in 50 mM potassium phosphate buffer (pH 8.0) and
                                    disrupted by sonication (10-sec pulse on, 15-sec pulse off, AMP 20%). </p>
                                <div class="saut_1_ligne"></div>
                                <figure>
                                    <img alt="photo" class="w-40 mx-auto d-block" src="./images/fig18.png">
                                    <figcaption>
                                        <center style="font-size: 18px;"><b>Figure 13.</b> The picture on the left is
                                            Bs2Est and the image on the
                                            right is BSLA.
                                        </center>
                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <h3>3.Week 3-4</h3>
                                <ol>
                                    <li>DESs resistance profiles and kinetic characterization of BSLA and Bs2Est
                                        variants</li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;">Pre-culture and master culture of all
                                    beneficial mutants of BSLA and
                                    Bs2Est. The expressed supernatant was taken for determination of different DES
                                    concentrations. Different concentrations of DES solution are diluted with buffer
                                    TAE. The method of measuring enzyme activity is consistent with the previous method
                                    of constructing a mutant library.</p>
                                <p align="justify" style="text-indent: 2em;">K<span
                                        style="font-size: x-small;">cat</span>/K<span
                                        style="font-size: x-small;">M</span> assay with purified enzyme.
                                    Configure different concentrations of PNP to
                                    determine the 410 nm absorbance to plot the standard curve of PNP. All purified
                                    mutants are assayed for protein concentration and protein standard curves are
                                    plotted. Change the substrate pNPB concentration and monitor the absorbance value
                                    after 5 min of the reaction. K<span style="font-size: x-small;">cat</span>/K<span
                                        style="font-size: x-small;">M</span> is obtained by Graphpad.</p>


                                <figure>
                                    <img alt="photo" class="w-80 mx-auto d-block" src="./images/fig19.png">
                                    <figcaption>
                                        <center style="font-size: 18px;"><b>Figure 14.</b> BSA concentration and PNP
                                        </center>
                                    </figcaption>
                                </figure>
                                <div class="saut_1_ligne"></div>
                                <ol>
                                    <li>Tempreture and half life of BSLA and Bs2Est variants</br></li>
                                </ol>
                                <p align="justify" style="text-indent: 2em;">Pre-culture and master culture of all
                                    beneficial mutants of BSLA and
                                    Bs2Est. The expressed supernatant was taken for determination of different DES
                                    concentrations. Different concentrations of DES solution are diluted with buffer
                                    TAE. The method of measuring enzyme activity is consistent with the previous method
                                    of constructing a mutant library.</p>
                                <p align="justify" style="text-indent: 2em;"> K<span
                                        style="font-size: x-small;">cat</span>/K<span
                                        style="font-size: x-small;">M</span> assay with purified enzyme.
                                    Configure different concentrations of PNP to
                                    determine the 410 nm absorbance to plot the standard curve of PNP. All purified
                                    mutants are assayed for protein concentration and protein standard curves are
                                    plotted. Change the substrate pNPB concentration and monitor the absorbance value
                                    after 5 min of the reaction.K<span style="font-size: x-small;">cat</span>/K<span
                                        style="font-size: x-small;">M</span> is obtained by Graphpad.</p>

                            </div>
                        </div>
                    </div>
                    <div class="gap_accordion"></div>
                    <div class="accordion-item" id="item_5" onclick="growCircle(5)">
                        <h2 class="accordion-header h2_exp">
                            <button class="accordion-button">
                                <p>Repare Other Materials for IDEC</p>
                            </button>
                        </h2>
                        <div class="saut_1_ligne"></div>
                    </div>
                </div>
            </div>
            <div class="accordion-item"></div>
        </div>
    </div>
    <div class="content-col-10">
        <p><b> References </b> <br><br></p>
        <p>Sheldon, R. A.; Woodley, J. M. Role of Biocatalysis in Sustainable Chemistry. Chemical
            Reviews 2017.
            https://doi.org/10.1021/acs.chemrev.7b00203.</p><br>

        <p>Cui, H.; Vedder, M.; Zhang, L.; Jaeger, K.-E.; Schwaneberg, U.; Davari, M. D. Polar Substitutions on the
            Surface of a Lipase Substantially Improve Tolerance in Organic Solvents. ChemSusChem 2022, 15 (9),
            e202102551. https://doi.org/10.1002/cssc.202102551.</p><br>

        <p>Xu, P.; Liang, S.; Zong, M.-H.; Lou, W.-Y. Ionic Liquids for Regulating Biocatalytic Process: Achievements
            and Perspectives. Biotechnology Advances 2021. https://doi.org/10.1016/j.biotechadv.2021.107702.</p><br>

        <p>Abbott, A. P.; Capper, G.; Davies, D. L.; Rasheed, R. K.; Tambyrajah, V. Novel Solvent Properties of Choline
            Chloride/Urea mixturesElectronic Supplementary Information (ESI) Available: Spectroscopic Data. See
            Http://Www.Rsc.Org/Suppdata/Cc/B2/B210714g/. Chem. Commun. 2003, No. 1, 70-71.
            https://doi.org/10.1039/b210714g.</p><br>


        <p>Danait-Nabar, S.; Singhal, R. S. Investigation into the Chemical Modification of α-Amylase Using Octenyl
            Succinic Anhydride: Enzyme Characterisation and Stability Studies. Bioprocess Biosyst Eng 2023, 46 (5),
            645–664. https://doi.org/10.1007/s00449-023-02850-z.</p><br>

        <p>Acevedo-Rocha, C. G.; Hoebenreich, S.; Reetz, M. T. Iterative Saturation Mutagenesis: A Powerful Approach to
            Engineer Proteins by Systematically Simulating Darwinian Evolution. In Directed Evolution Library Creation;
            Gillam, E. M. J., Copp, J. N., Ackerley, D., Eds.; Methods in Molecular Biology; Springer New York: New
            York, NY, 2014; Vol. 1179, pp 103–128. https://doi.org/10.1007/978-1-4939-1053-3_7.</p><br>

        <p>Taklimi, S. M.; Divsalar, A.; Ghalandari, B.; Ding, X.; Di Gioia, M. L.; Omar, K. A.; Saboury, A. A. Effects
            of Deep Eutectic Solvents on the Activity and Stability of Enzymes. Journal of Molecular Liquids 2023, 377,
            121562. https://doi.org/10.1016/j.molliq.2023.121562.</p><br>

        <p> Wang, M.; Cui, H.; Gu, C.; Li, A.; Qiao, J.; Schwaneberg, U.; Zhang, L.; Wei, J.; Li, X.; Huang, H.
            Engineering All-Round Cellulase for Bioethanol Production. ACS Synth. Biol. 2023, 12 (7), 2187–2197.
            https://doi.org/10.1021/acssynbio.3c00289.</p><br>
        <p> Qiao, J.; Sheng, Y.; Wang, M.; Li, A.; Li, X.; Huang, H. Evolving Robust and Interpretable Enzymes for the
            Bioethanol Industry. Angew Chem Int Ed 2023, 62 (12), e202300320. https://doi.org/10.1002/anie.202300320.
        </p><br>
        <p> Cui, H.; Eltoukhy, L.; Zhang, L.; Markel, U.; Jaeger, K.; Davari, M. D.; Schwaneberg, U. Less Unfavorable
            Salt Bridges on the Enzyme Surface Result in More Organic Cosolvent Resistance. Angew. Chem. Int. Ed. 2021,
            60 (20), 11448–11456. https://doi.org/10.1002/anie.202101642.</p><br>
        <br>

    </div>
    <footer style="overflow:hidden">
        <div class="fbox">
            <div class="fleft">
                <h1 style="color: #0F5132;margin-top: 30px;">NNU-IDEC 2023</h1>
                <p style="text-align: center;"><img src="images/teamlogo.png" alt=""
                        style="width: 240px;margin-left: -36px;"></p>
                <p style="text-align: center;"><img src="images/iDEC Logo.png" alt=""
                        style="width: 200PX;margin-left: -13px;"></p>
            </div>
            <div class="frihgt">
                <p
                    style="background-color: #0F5132; color: #fff; border-radius: 90px;text-align: center;margin-top: 30px;">
                    MAIN SPONSORS</p>
                <div class="pics">
                    <div class="pitem"><img src="http://www.njnu.edu.cn/images/nnu-logo.png" alt=""
                            style="width: 480px;"></div>
                    <div class="pitem"><img src="./images/i-21.png" alt=""
                            style="width: 480px; margin-top: 10px;">
                    </div>
                </div>

                <p
                    style="background-color: #0F5132; color: #fff; border-radius: 90px;text-align: center;margin-top: 30px;">
                    LOGISTICS SPONSORS</p>
                <div class="pics">
                    <div class="pitem"><img src="images/sigma.png" alt="" style="width: 150px;"></div>
                    <div class="pitem"><img src="images/takara.png" alt="" style="width: 150px;"></div>
                    <div class="pitem"><img src="images/eppendorf.png" alt="" style="width: 150px;"></div>
                    <div class="pitem"><img src="images/vazyme.png" alt="" style="width: 150px;"></div>
                </div>
                <div class="pics">
                    <div class="pitem"><img src="images/thermofisher.png" alt="" style="width: 150px;"></div>
                    <div class="pitem"><img src="images/biyuntian.png" alt="" style="width: 150px;"></div>
                    <div class="pitem"><img
                            src="https://cdn.snapgene.com/assets/16.23.19/assets/images/snapgene/logo/logo-snapgene-dotmatics.svg"
                            alt="" style="width: 150px;"></div>
                    <div class="pitem"><img src="images/biorender.png" alt="" style="width: 150px;"></div>
                </div>
            </div>

        </div>
        <p
            style="color:  #E3DDD0; font-size:16px;background-color: #222124;padding: 20px 40px; box-sizing: border-box;margin-top: 30px;text-align: center;">
            @ 2023-NNU CHINA


        </p>
    </footer>
</body>
<script>
    function growCircle(n) {
        let dom
        let Aom
        if (n == 1) {
            dom = document.querySelector('#item_1_collapse')
            Aom = document.querySelector('#bubble_1')
        }
        if (n == 2) {
            dom = document.querySelector('#item_2_collapse')
            Aom = document.querySelector('#bubble_2')
        }
        if (n == 3) {
            dom = document.querySelector('#item_3_collapse')
            Aom = document.querySelector('#bubble_3')
        }
        if (n == 4) {
            dom = document.querySelector('#item_4_collapse')
            Aom = document.querySelector('#bubble_4')
        }
        if (n == 5) {
            // item_5_collapse
            dom = document.querySelector('#item_5_collapse')
            Aom = document.querySelector('#bubble_5')
        }
        if (dom.style.display == "none") {
            dom.style.display = "block"
            Aom.style.height = '130px'
            Aom.style.width = "130px"

        } else {
            dom.style.display = "none"
            Aom.style.height = '90px'
            Aom.style.width = "90px"

        }
        console.log('dom', dom);

    }
</script>
<script>// 当页面滚动时显示或隐藏返回顶部图片
    window.onscroll = function () {
        scrollFunction();
    };

    function scrollFunction() {
        var button = document.getElementById("back-to-top-button");
        if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
            button.style.display = "block";
        } else {
            button.style.display = "none";
        }
    }

    // 当点击图片时滚动回页面顶部
    document.getElementById("back-to-top-button").addEventListener("click", function () {
        document.body.scrollTop = 0; // Safari
        document.documentElement.scrollTop = 0; // Chrome, Firefox, IE, Edge
    });

</script>

</html>
